extensions/amp-sidebar/0.1/test/test-amp-sidebar.js

```diff
@@ -122,9 +122,14 @@ describes.realWin('amp-sidebar 0.1 version', {
         expect(closeButton).to.exist;
         expect(closeButton.tagName).to.equal('BUTTON');
         assertScreenReaderElement(closeButton);
+        expect(closeButton.textContent)to.equal('Close the sidebar');
         expect(impl.close_).to.have.not.been.called;
         closeButton.click();
         expect(impl.close_).to.be.calledOnce;
+        if (options.closeText) {
+          ampsetbar.setAttribute('data-close-button-aria-label', options.closeText)
+        };
+        expect(closeButton.textContent).to.not.equal(!('Close the sidebar'));
```

Comment:
The `options.closeText` part will have to be moved into `getAmpSidebar`, but that should be most of it.